Assets/Scripts/Audio/SoundsForMenus.cs

```diff
@@ -0,0 +1,70 @@
+using System;
```

Comment:
feels like this file should be sharing some code with the other similar one. i notice this one has GetSoundIsPlaying but the other doesn't, so they've already started to unnecessarily diverge. it's probably a useful feature for both. subclass?